ppr-ui/src/components/mhrRegistration/HomeOwners/AddHomeOwnerPerson.vue

```diff
@@ -0,0 +1,219 @@
+                  v-model="individualName.last"
+                  filled
+                  :rules="lastNameRules"
+                  label="Last Name"
+                  data-test-id="last-name"
+                />
+              </v-col>
+            </v-row>
+
+            <label class="generic-label" for="manufacturer-name">
+              Phone Number
+            </label>
+            <v-row>
+              <v-col cols="6">
+                <v-text-field
+                  id="phone-number"
+                  v-model="phoneNumber"
+                  filled
+                  :rules="phoneNumberRules"
+                  label="Phone Number"
+                  data-test-id="phone-number"
+                />
+              </v-col>
+              <v-col cols="6">
+                <v-text-field
+                  id="phone-ext"
+                  v-model="phoneExtension"
+                  filled
+                  :rules="maxLength(5, true)"
+                  label="Extension (Optional)"
+                  data-test-id="phone-ext"
+                />
+              </v-col>
+            </v-row>
+
+            <label class="generic-label" for="manufacturer-name">
+              Mailing Address
+            </label>
+
+            <base-address
+              :editing="true"
+              :schema="{ ...addressSchema }"
+              v-model="address"
+              :triggerErrors="triggerAddressErrors"
+              @valid="isAddressFormValid = $event"
+              class="mt-2"
+            />
+          </v-form>
+
+          <v-row>
+            <v-col cols="6">
+              <v-btn
+                outlined
+                color="error"
+                class="remove-btn"
+                :disabled="isAddingHomeOwner"
+                :ripple="false"
+                @click="remove()"
+              >
+                Remove
+              </v-btn>
+            </v-col>
+            <v-col cols="3">
+              <v-btn :ripple="false" width="100%" @click="cancel()">
+                Cancel
+              </v-btn>
+            </v-col>
+            <v-col cols="3">
+              <v-btn
+                color="primary"
+                :ripple="false"
+                width="100%"
+                @click="done()"
+              >
+                Done
+              </v-btn>
+            </v-col>
+          </v-row>
+        </v-col>
+      </v-row>
+    </v-card>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent, reactive, ref, toRefs } from '@vue/composition-api'
+import { useInputRules } from '@/composables/useInputRules'
+import { BaseAddress } from '@/composables/address'
+import { PartyAddressSchema } from '@/schemas'
+import { focusOnFirstError } from '@/utils'
+/* eslint-disable no-unused-vars */
+import { MhrRegistrationHomeOwnersIF } from '@/interfaces/mhr-registration-interfaces'
+/* eslint-enable no-unused-vars */
+
+export default defineComponent({
+  name: 'AddHomeOwnerPerson',
```

Comment:
Technically this is our `AddEdit` form, perhaps we could update the component name.

To that point, besides the Name portion of this form there is a lot of duplication between HomeOwnerPerson and HomeOwnerOrganization. 

Couldn't we just create one component, `AddEditHomeOwner`?

Reply:
Refactored to have one component for adding a Person or Org.